lib/OpenQA/WebAPI/Plugin/Helpers.pm

```diff
@@ -88,9 +88,7 @@ sub register {
         current_job_group => sub {
             my ($c) = @_;
 
-            my $test = $c->param('testid');
-
-            if ($test) {
+            if ($c->param('testid') || ($c->req->url->path =~ qr/.*\/tests\/latest/)) {
```

Comment:
Rather than escaping the `/` in regex with `/` as a delimiter you should use a different delimiter, e.g. `qr{.*/tests/latest}`